run_dir/static/js/suggestion_box.js

```diff
@@ -66,21 +69,18 @@ $.validator.setDefaults({
 //Generate alert when POSTing and refresh suggestions table
 $("#suggestionForm").validate({
   submitHandler: function(form) {
-    var spinnerText = "<div class='test_padding'> \
-                        <span class='icon-refresh glyphicon-refresh-animate'></span>  Processing your suggestion... \
-                      </div>"
-    $('#modalBody').html(spinnerText);
+    var spinnerText = "<div style=\"text-align:center; margin:20px 0;\"><span class=\"glyphicon glyphicon-refresh glyphicon-spin\"></span>  Processing your suggestion..</div>"
```

Comment:
Minor thing, but backslashed quote marks bug me. Why not use single quotes for the string instead?